spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/reactor/SleuthDecorator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.instrument.reactor;
+
+/**
+ * Only for internal use. Marker interface for Publisher and Subscriber that propagate
+ * brave context into execution Thread.
+ */
+@Deprecated
+public interface SleuthDecorator {
```

Comment:
At first I thought this was something else. Is it important to say Sleuth in the name? (wondering if the package is somehow missing in debug tools). If not, I would call it `TraceContextXXX` as this is similar to brave's rxjava2 propagation lib (except we didn't do operator fusion style dedupe there). The name `TraceContextXXX` is more precise as for example, this does not mutate spans or other features sleuth might do.

Reply:
`TraceContextPropagator` - is it fine?